IPython/core/pylabtools.py

```diff
@@ -261,6 +262,8 @@ def find_gui_and_backend(gui=None, gui_select=None):
     if gui and gui != 'auto':
         # select backend based on requested gui
         backend = backends[gui]
+        if gui == 'agg':
```

Comment:
Since `'agg'` is an addition to the function, it would be helpful to add it to the function's docstring parameters for `gui`.

> We could modify something either here or in ipykernel to make it easier to specify "this backend needs no GUI integration", which would be true for nbagg, inline, agg.

Good suggestion.